beacon-chain/main.go

```diff
@@ -163,6 +173,82 @@ func main() {
 	}
 }
 
+func loadChainConfigFile(chainConfigFileName string) {
```

Comment:
why is this in `main.go` instead of the `node` package ?

Reply:
moved